data/data/gcp/network/firewall-control.tf

```diff
@@ -36,6 +36,19 @@ resource "google_compute_firewall" "master_ingress_https" {
   target_tags   = ["${var.cluster_id}-master"]
 }
 
+resource "google_compute_firewall" "master_ingress_https_from_external_lb" {
+  name    = "${var.cluster_id}-master-ingress-https-from-external-lb"
+  network = google_compute_network.cluster_network.self_link
+
+  allow {
+    protocol = "tcp"
+    ports    = ["6443"]
+  }
+
+  source_ranges = ["${google_compute_forwarding_rule.api_external.ip_address}/32"]
```

Comment:
looking at https://cloud.google.com/load-balancing/docs/health-checks#fw-netlb

it looks like the ingress needs to be allowed from all the entire list of range?? https://cloud.google.com/load-balancing/docs/health-checks#fw-netlb

Reply:
from https://cloud.google.com/load-balancing/docs/health-check-concepts#multiple-probers
```
GCP sends health check probes from multiple redundant systems from the appropriate source IP ranges. No single probe system is responsible for all of the probes. Multiple systems issue probes simultaneously so that failure of one does not cause GCP to lose track of VM health states.
```

Reply:
Oh, I misunderstood those to be example IPs... I'll update now.

Reply:
Updated to allow of the ranges.